quinn-udp/src/unix.rs

```diff
@@ -151,10 +151,6 @@ fn init(io: SockRef<'_>) -> io::Result<()> {
         set_socket_option(&*io, libc::IPPROTO_IPV6, libc::IPV6_DONTFRAG, OPTION_ON)?;
     }
 
-    if !is_ipv4 {
-        set_socket_option(&*io, libc::IPPROTO_IPV6, libc::IPV6_RECVTCLASS, OPTION_ON)?;
```

Comment:
So is this because it's redundant with the `IPV6_RECVPKTINFO` call? Are the `RECVPKTINFO` and `RECVTCLASS` options documented to be redundant? Is that something we should add some comments on to the remaining call?

Reply:
Nothing so subtle -- it's redundant because we make exactly the same call just above on line 147.